cli/cmd/root.go

```diff
@@ -315,6 +327,10 @@ func (options *proxyConfigOptions) validate() error {
 		return fmt.Errorf("--tls must be blank or set to \"%s\"", optionalTLS)
 	}
 
+	if !validProxyLogLevel.MatchString(options.proxyLogLevel) {
+		return fmt.Errorf("%s is not a valid proxy log level", options.proxyLogLevel)
```

Comment:
Perhaps this error should provide a link to https://docs.rs/env_logger/0.6.0/env_logger/#enabling-logging as a reference?